corehq/messaging/scheduling/forms.py

```diff
@@ -471,7 +479,7 @@ def compute_initial(domain, content):
             result['sms_callback_intervals'] = ', '.join(str(i) for i in content.reminder_intervals)
         else:
             raise TypeError("Unexpected content type: %s" % type(content))
-
+        result['should_run_rule'] = "true"
```

Comment:
why is this needed?

Reply:
It's to initialize the value to "true". I guess it can be achieved by default value also.